etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -1,5 +1,37 @@
 # Generated by confd
 filter calico_export_to_bgp_peers {
+  if bgp_passthru() then accept;
+  calico_aggr();
+{{- $static_key := "/staticroutes"}}
+{{- if ls $static_key}}
+
+  # Export static routes.
```

Comment:
Looks like this logic is duplicated now in export_to_bgp_peers and export_to_bgp_mesh. Probably worth using a common function - e.g., `if static_route_accept() then accept;` WDYT?

Reply:
Sure, will do.